### DISCUSSION TITLE
Why does `/node/memory` not correspond roughly to `NumHeapPages`?

### DISCUSSION THREAD

**Discussion by letmaik** [microsoft/CCF#3709] [Q&A] [answered]
In `oe_sign.conf` I have `NumHeapPages=50000`, which is ~390MB.
The `/node/memory` endpoint returns `"max_total_heap_size": 204800000` which is ~195MB.
Why is there such a big difference?

**Comment by wintersteiger** [comment, marked as answer]
IIRC, a heap page in OE is 4096 bytes, so 50k pages are exactly 204800000 bytes. Our endpoint just returns whatever OE reports and neither of us is adjusting this for any other factors, I think.

**Reply by letmaik** [reply]
You're right, I must have assumed 8 KB heap page size for some reason.
